framework/source/class/qx/io/remote/Rpc.js

```diff
@@ -568,7 +568,18 @@ qx.Class.define("qx.io.remote.Rpc",
             break;
 
           case 1: // async with handler function
-            handler(result, ex, id);
+            try
+            {
+              handler(result, ex, id);
+            }
+            catch(e)
+            {
+              console.error(
+                "rpc handler threw an error:" +
+                  " id=" + id +
+                  " result=" + JSON.stringify(result) +
+                  " ex=" + JSON.stringify(ex));
```

Comment:
Ok. So it should be a decision to do a large-scale clean-up of qx.lang.Json, but for now, I'll make that change here and defer the large-scale clean-up for another time.